modules/analytics/src/views/full/index.tsx

```diff
@@ -338,13 +388,19 @@ const Analytics: FC<any> = ({ bp }) => {
           itemLimit={10}
           className={cx(style.genericMetric, style.half, style.list)}
         />
-        <ItemsList
-          name={lang.tr('module.analytics.mostAskedQuestions')}
-          items={getTopItems('msg_sent_qna_count', 'qna')}
-          itemLimit={10}
-          hasTooltip
-          className={cx(style.genericMetric, style.half, style.list)}
-        />
+        {!_.isEmpty(state.qnaQuestions) && (
+          <ItemsList
+            name={lang.tr('module.analytics.mostAskedQuestions')}
+            items={getTopItems(
+              'msg_sent_qna_count',
+              'qna',
+              id => state.qnaQuestions[id],
+              // Filter out QnA metrics without submetric (legacy)
+              metric => metric.subMetric
```

Comment:
Can you add your 2 properties as an object instead ? it's not clear what those just by looking here

Reply:
Done in 13eb5c6